src/main/java/com/couchbase/lite/replicator/PullerInternal.java

```diff
@@ -515,8 +515,6 @@ public void insertDownloads(List<RevisionInternal> downloads) {
                     }
                 }
 
-                if(rev.getBody() != null) rev.getBody().release();
-
```

Comment:
Will removing this line cause the memory regression issue during the pull replicating?